src/components/DatePicker.ts

```diff
@@ -14,6 +14,13 @@ class DatePicker extends FormField {
     await this.textbox.fill(formatDate(date));
     await this.page.keyboard.press('Enter');
   }
+
+  async fillWithFormat(date: Date, format: string) {
+    await this.page.waitForTimeout(1000);
```

Comment:
Please add a "todo" or "fixme" comment here, that this is a temporary solution until we figure out the sluggishness of the date picker